docs/Operators.md

```diff
@@ -19681,6 +19682,282 @@ This version of the operator has been available since version 11 of the default
 </dl>
 
 
+### <a name="SequenceMap"></a><a name="sequencemap">**SequenceMap**</a>
+
+  Applies a sub-graph to each sample in the input sequence(s).
+
+  Inputs can be either tensors or sequences, with the exception of the first input which must
+  be a sequence. The length of the first input sequence will determine the number of samples in the
+  outputs. Any other sequence inputs should have the same number of samples. The number of inputs
+  and outputs, should match the one of the subgraph.
+
+  For each element in the output, a sample will be extracted from the input sequence(s) and the sub-graph
+  will be applied to it. The outputs will contain the outputs of the sub-graph for each sample.
```

Comment:
Please specify that an output at position X in the sequence is computed from applying the the subgraph from the input at the same position X in the input sequence.

I would add the following. Users cannot expect any specific ordering in which each subgraph is computed.

Reply:
@AlexandreEichenberger That's a great point, thank you. I've updated the text. Could you please check that it makes sense?